pods-beaver-themer.php

```diff
@@ -80,6 +80,21 @@ function pods_beaver_admin_nag() {
 add_action( 'plugins_loaded', 'pods_beaver_admin_nag' );
 
 /**
+ * Enqueue assets
+ *
+ * @return void
+ *
+ * @since 1.1
+ */
+function pods_beaver_enqueue_assets() {
+    if ( FLBuilderModel::is_builder_active() ) {
```

Comment:
I don't *think* so... if you check FLBuilder::render_module_settings you will see how BB natively adds the settings JS and CSS for the module (ie settings.js and settings.css), and that is just script tags, not enqueue
<img width="867" alt="screenshot 2017-06-20 16 49 34" src="https://user-images.githubusercontent.com/8013101/27342594-85ac6ca4-55d8-11e7-8f68-3e6060c35a18.png">